ui/org.eclipse.pde.launching/src/org/eclipse/pde/internal/launching/launcher/LaunchValidationOperation.java

```diff
@@ -43,6 +43,11 @@ public class LaunchValidationOperation implements IWorkspaceRunnable {
 
 	private BundleValidationOperation fOperation;
 	protected final ILaunchConfiguration fLaunchConfiguration;
+
+	public ILaunchConfiguration getfLaunchConfiguration() {
```

Comment:
let's skip the `f` here and make it just `getLaunchConfiguration()`.
Note that since all of this is internal, a viable solution is to make the field `public final`.